decentraland/unity-renderer
# Fix: Green blockers should not be visible unless the scene is being loaded

## What does this PR change?

This PR fix 2 related issues:
1. the problem was green blockers appearing on edge parcels even when no content was being loaded. Now the green blockers appear on any empty or non-empty parcel only while the content is still being loaded.
2. the problem where the user could select from the settings panel a load radius of 1. This problem caged the user in a 1 by 1 square parcel, without any possibility to escape. Now the minimum radius is 2.
...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/not-loading-green-blockers
2. Set the loading radius in the settings to 2, to easily see green-blockers when exploring.
3. Explore parcels outside Genesis plaza to check if green blockers spawn correctly
4. Teleport to a remote area where there are a lot of empty scenes, and check if green-blockers appear only if there is something loading.
5. Try changing the radius from the setting, and see if you can select 1 anymore.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
